test/benchmark_http_client_test.cc

```diff
@@ -55,6 +55,22 @@ class BenchmarkClientHttpTest : public Test {
               {{":scheme", "http"}, {":method", "GET"}, {":path", "/"}, {":host", "localhost"}}));
       return std::make_unique<RequestImpl>(header);
     };
+    statistic_.connect_statistic = std::make_unique<StreamingStatistic>();
+    statistic_.response_statistic = std::make_unique<StreamingStatistic>();
+    statistic_.response_header_size_statistic = std::make_unique<StreamingStatistic>();
+    statistic_.response_body_size_statistic = std::make_unique<StreamingStatistic>();
+    statistic_.latency_1xx_statistic =
```

Comment:
Agree that `deliverHistogramToSinks` should not be tested here (it is already tested in [statistic_test.cc](https://github.com/envoyproxy/nighthawk/blob/0b23bbfde94ffa324940bec34ffaf9bba4f68272/test/statistic_test.cc#L417-L419)).

There is [NullStatistic](https://github.com/envoyproxy/nighthawk/blob/0b23bbfde94ffa324940bec34ffaf9bba4f68272/source/common/statistic_impl.h#L42), however it can not be used to test `BenchmarkClientHttpImpl::exportLatency()` meaningfully. So I changed all `latency_statistic` to be `StreamingStatistic`.